jobs/email-reminder/flags.json

```diff
@@ -0,0 +1,16 @@
+{
+    "flagValues": {
+        "enable-bc-ccc-ulc": false,
```

Comment:
found this flag in the code, so add it there as well https://github.com/bcgov/lear/blob/main/jobs/email-reminder/email_reminder.py#L146C6-L146C45